components/input/input.css

```diff
@@ -0,0 +1,62 @@
+.form{
+    border:solid var(--6C757D) ;
```

Comment:
Maintain space after :

Reply:
Ok I will fix the indentation thanks for Reviewing :)